### ISSUE TITLE
Endlessly "Updating Accessory"

### ISSUE THREAD

**Issue by mkrecek234** [ilcato/homebridge-Fibaro-HC2#63]
Homebridge is stuck while "Updating Accessory" for all my connected Fibaro devices. HomeKit also does not show HomeBridge as a device:

[5/13/2017, 1:34:08 PM] [FibaroHC2] Updating Accessory: Fenster rechts
[5/13/2017, 1:34:08 PM] [FibaroHC2] Updating Accessory: Fenster links
[5/13/2017, 1:34:09 PM] [FibaroHC2] Updating Accessory: Licht Flur
[5/13/2017, 1:34:09 PM] [FibaroHC2] Updating Accessory: Temperatur Flur
[5/13/2017, 1:34:09 PM] [FibaroHC2] Updating Accessory: Bewegungsmelder
[5/13/2017, 1:34:09 PM] [FibaroHC2] Updating Accessory: Rollo
[5/13/2017, 1:34:09 PM] [FibaroHC2] Updating Accessory: Espressomaschine
[5/13/2017, 1:34:09 PM] [FibaroHC2] Updating Accessory: Bewegungsmelder
[5/13/2017, 1:34:09 PM] [FibaroHC2] Updating Accessory: Tamper Terrassentür
[5/13/2017, 1:34:09 PM] [FibaroHC2] Updating Accessory: Terrassentür
[5/13/2017, 1:34:09 PM] [FibaroHC2] Updating Accessory: Temperatur Wohnen
[5/13/2017, 1:34:09 PM] [FibaroHC2] Updating Accessory: Licht Wohnen
[5/13/2017, 1:34:09 PM] [FibaroHC2] Updating Accessory: Tamper Balkontür SZ
[5/13/2017, 1:34:09 PM] [FibaroHC2] Updating Accessory: Balkontür
[5/13/2017, 1:34:09 PM] [FibaroHC2] Updating Accessory: Rollo links
[5/13/2017, 1:34:09 PM] [FibaroHC2] Updating Accessory: Temperatur
[5/13/2017, 1:34:09 PM] [FibaroHC2] Updating Accessory: Bewegungsmelder
[5/13/2017, 1:34:09 PM] [FibaroHC2] Updating Accessory: Rollo rechts
[5/13/2017, 1:34:09 PM] [FibaroHC2] Updating Accessory: Licht Eingang

I updated to the latest version of home bridge-Fibaro-HC2...

**Comment by ilcato** [comment]
I see nothing wrong with the log (the updating sequence is normal, maybe I will change the expression to Updated accessory: ...).
Is this the first time you execute homebridge. Was it working ?

**Comment by mkrecek234** [comment]
Hi,

It used to work and I could add all accessory within HomeKit.
After some time all accessories within HomeKit showed „Updating…“ and were not functioning anymore, and also the log data of Homebridge showed this.

I decided to remove Homebridge from HomeKit and when trying to re-add HomeKit could not find Homebridge anymore.
Before, the behaviour was different: When launching Homebridge, all accessories were added, and the code showed up, but no message list with „updating“.

Right now, I cannot use Homebridge at all. Any hint?
Best regards and thank you for the quick response!

Michael
